data/types/blood/glucose/glucose.go

```diff
@@ -33,3 +36,75 @@ func (g *Glucose) Normalize(normalizer data.Normalizer) {
 		g.Value = dataBloodGlucose.NormalizeValueForUnits(g.Value, units)
 	}
 }
+
+func roundToEvenWithDecimalPlaces(v float64, decimals int) float64 {
+	if decimals < 1 {
+		return math.RoundToEven(v)
+	}
+	coef := math.Pow(10, float64(decimals))
+	return math.RoundToEven(v*coef) / coef
+}
+
+type Classification string
+
+const (
+	ClassificationInvalid Classification = "invalid"
+
+	VeryLow       = "very low"
+	Low           = "low"
+	OnTarget      = "on target"
+	High          = "high"
+	VeryHigh      = "very high"
+	ExtremelyHigh = "extremely high"
+)
+
+type classificationThreshold struct {
+	Name      Classification
+	Value     float64
+	Inclusive bool
+}
+
+type Classifier []classificationThreshold
+
+func (c Classifier) Classify(g *Glucose) (Classification, error) {
+	normalized, err := dataBloodGlucose.NormalizeValueForUnitsSafer(g.Value, g.Units)
+	if err != nil {
+		return ClassificationInvalid, errors.Wrap(err, "unable to classify")
+	}
+	// Rounded values are used for all classifications. To not do so risks introducing
+	// inconsistency between frontend, backend, and/or other reports. See BACK-3800 for
+	// details.
+	rounded := roundToEvenWithDecimalPlaces(normalized, 1)
```

Comment:
Hmm, if you are going to use the datum units (above `g.Units`), then I don't think you should hard-code the `1` here (which is, by extension, hard-coding mmol/L).

Reply:
> Hmm, if you are going to use the datum units (above `g.Units`), then I don't think you should hard-code the `1` here (which is, by extension, hard-coding mmol/L).

I don't understand why not? NormalizeValueForUnits can only return units in mmol/L. It doesn't return any other units.

Reply:
Ah, okay, makes sense.